### ISSUE TITLE
npm dist js version only allows create with default.create

### ISSUE THREAD

**Issue by Freundschaft** [caseyWebb/le-challenge-redis#4]
when I require the npm version in nodejs I have to use

`let leHttpChallenge = require('le-challenge-redis').default.create({`

is this intended behavior?

**Comment by caseyWebb** [comment]
I just published 1.0.0 using named exports for better commonjs/es2015 interop. I don't like publishing things with default exports usually for that reason.

try this

```javascript
const { LetsEncryptRedisChallenge } = require('le-challenge-redis')

const leHttpChallenge = LetsEncryptRedisChallenge.create({
  ...
})
```

**Comment by Freundschaft** [comment]
looking good, thanks!
